src/zc/form/browser/widgetapi.py

```diff
@@ -223,7 +223,7 @@ class BaseVocabularyDisplay(BaseVocabularyWidget):
     _display = True
 
     def render(self, value):
-        if value in (NullValue, None):
+        if value is None:
```

Comment:
This looks like a bugfix for a NameError rather than a cleanup.

(It's also an indication that this method is not covered by tests.)

Reply:
Done. I plan to have a deeper look at the test coverage in the next steps.